app/src/androidTest/java/org/mozilla/fenix/ui/SmokeTest.kt

```diff
@@ -1055,6 +1055,33 @@ class SmokeTest {
         }
     }
 
+    @Test
+    fun selectTabsButtonVisibilityTest() {
+        homeScreen {
+        }.dismissOnboarding()
+
+        val firstWebPage = TestAssetHelper.getGenericAsset(mockWebServer, 1)
+        val secondWebPage = TestAssetHelper.getGenericAsset(mockWebServer, 2)
+
+        navigationToolbar {
+        }.enterURLAndEnterToBrowser(firstWebPage.url) {
+            mDevice.waitForIdle()
+        }.openTabDrawer {
+        }.openNewTab {
+        }.submitQuery(secondWebPage.url.toString()) {
+            mDevice.waitForIdle()
+        }.openTabDrawer {
+        }.toggleToPrivateTabs {
+        }.openNewTab {
+        }.dismissSearchBar { }
+
+        homeScreen { }

```

Comment:
nit: Maybe we can have the same formatting as the above here?
as in 

```suggestion
        homeScreen {
        }.openTabDrawer {

```

Reply:
Hello @Mugurell 
I really appreciate your feedback and help! I'll fix all the remarks.